package.json

```diff
@@ -69,7 +69,7 @@
   "dependencies": {
     "Base64": "0.3.0",
     "core-js": "^3.6.5",
-    "cross-fetch": "^3.0.0",
+    "cross-fetch": "^3.0.0-beta.9",
```

Comment:
Is this right? The latest version is `3.0.6` which seems to be what this resolves to in the `yarn.lock`

Reply:
Thanks - this does not look right.
`3.0.0-beta.9` is a suggested version of `node-fetch`, not `cross-fetch`
<img width="714" alt="Screenshot 2021-02-03 at 13 26 35" src="https://user-images.githubusercontent.com/71440851/106741056-e2ce6e00-6623-11eb-907b-8caed59e454f.png">